### DISCUSSION TITLE
Release 提供Windows/Arm64支持

### DISCUSSION THREAD

**Discussion by Gaojianli** [v2fly/v2ray-core#1208] [Ideas]
随着go1.17的发布，[go正式提供了arm64的windows支持](https://golang.google.cn/doc/go1.17#windows)
经测试，直接使用GOARCH=arm64的环境变量即可成功编译v2ray
![image](https://user-images.githubusercontent.com/19235246/129870135-6cb56b76-3017-4e0a-a97a-8674901c76cc.png)
那么，官方release是否也可加入arm64 windows的支持？

**Comment by Loyalsoldier** [comment]
会有的
